### PR TITLE
Fix bakta running only for one sample

### LINKED ISSUE #171: Bakta running only for one sample
### Description of the bug

When running `bacass` for multiple samples, `bakta` output is produced only for one of them.
Execution trace and sw version are included in zip. (no `.nextflow.log` for this one, sorry)

### Command used and terminal output

_No response_

### Relevant files

[pipeline_info.zip](https://github.com/user-attachments/files/17203426/pipeline_info.zip)

### System information

_No response_

### PR DESCRIPTION
**Pull request by Daniel-VM**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bacass/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/bacass _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

## PR Description

Fixed bakta that was running for one sample only. 

Closes #171

### CODE REVIEW

**Comment by Daniel-VM** [comment]
I ran a few tests and found that when providing a local path to the Bakta database, the workflow processes only a single sample. However, when I set the workflow to download the database (`--baktadb_download true`), it processes all samples as expected. I'm not sure why this happens, but the difference seems related to how the database is accessed locally versus being downloaded.

Do you think I should try to find a more suitable fix?

**Comment by Daniel-VM** [comment]
After this change, all samples ran bakta using both both database options (providing a local path and downloading it) .

**Comment by Daniel-VM** [comment]
Wait, I just found the issue 😅 .

File: subworkflows/local/bakta_dbdownload_run.nf [hunk 1]
```diff
@@ -25,16 +25,14 @@ workflow BAKTA_DBDOWNLOAD_RUN {
 
             // MODULE: untar database
             UNTAR( ch_baktadb_tar )
-            ch_baktadb      = UNTAR.out.untar.map{ meta, db -> db }
+            ch_path_baktadb = UNTAR.out.untar.map{ meta, db -> db }
             ch_versions     = ch_versions.mix(UNTAR.out.versions)
-        } else {
-            ch_baktadb      = Channel.from(ch_path_baktadb).map{ db -> db }
```
Comment (Daniel-VM) [inline subworkflows/local/bakta_dbdownload_run.nf:31]: `ch_path_baktadb` is already a channel. Re-creating it to assign `ch_bakta` caused the unusual behavior of processing only a single sample.

Comment (jasmezz) [reply subworkflows/local/bakta_dbdownload_run.nf:31]: I see! I don't have to check how we did it in funcscan then, you already found the culprit ;)

**Review by jasmezz — APPROVED** [review]
LGTM now :rocket:

**Comment by Daniel-VM** [comment]
Thanks @jasmezz  :)

**Pull request merged**